### PR TITLE
Update some dependency information in README

### PR DESCRIPTION
**Pull request by joto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,14 +84,15 @@ On a Debian or Ubuntu system, this can be done with:
...
+  nlohmann-json3-dev pyosmium
 ```
 
 On a Fedora system, use
 
 ```sh
 sudo dnf install cmake make gcc-c++ boost-devel expat-devel zlib-devel \
-  potrace-devel cimg-devel \
+  potrace-devel cimg-devel json-devel \
   bzip2-devel postgresql-devel proj-devel proj-epsg lua-devel pandoc
```
Comment (lonvia) [inline README.md:96]: At least Fedora should have pyosmium, too. Not sure about CentOS but we might consider removing that completely anyway, given the state of affairs.

Comment (joto) [reply README.md:96]: There are two packages in Fedora: pyosmium and python3-osmium. Not sure which one to install.

@tomhughes ?

**Comment by tomhughes** [comment]
The source package is `pyosmium` but the binary package that you need to install to use it is `python3-osmium`.

File: README.md (hunk 1 above)
Comment (tomhughes) [reply README.md:96]: The source package is `pyosmium` but the binary package that you need to install to use it is `python3-osmium`.

Comment (joto) [reply README.md:96]: I have updated the commit accordingly.

**Pull request merged**